monitoring/README.md

```diff
@@ -49,26 +89,51 @@ wds1-ks-transport-controller   64m
 b) ITS space:
 
 ```bash
-$ ./configure-metrics-its.sh
+./configure-monitoring-its.sh 
 ```
 
-If you have more than one ITS, you can run the above script with the `--space-name` flag. For example: `./configure-metrics-its.sh --space-name its2`
+If you have more than one ITS, you can run the above script with the `--space-name` flag. For example: `./configure-monitoring-its.sh --space-name its2`
 
 Optionally, check the prometheus service monitor objects for the ITS space:
 
 ```bash
-$ kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack" | grep its1*
+kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack" | grep its1*
+```
+
+Output:
+```bash
 its1-apiserver                 64m
 its1-status-addon-controller   64m
 ```
 
+c) Workload execution cluster (WEC):
+
+Run the following command for every WEC: `$ ./configure-monitoring-wec.sh --wec-cluster-context <wec-ctx-name>`.
+
+For example:
+```bash
+./configure-monitoring-wec.sh --wec-cluster-context cluster1
+```
+
+Optionally, check the prometheus service monitor object:
+
+```bash
+kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack"
+```
+
+Output:
+```bash
+NAME                    AGE
+cluster1-status-agent   47s
+```
+
 
 #### 3. Connect to Grafana UI: 
 
 a) Port-forward Grafana to localhost, by using the kubectl command:
 
 ```bash
-$ kubectl port-forward -n ks-monitoring svc/grafana 3000:80
+kubectl port-forward -n ks-monitoring svc/grafana 3000:80
 ```
 
 b) Check Prometheus and Pyroscope data source configuration: in a browser, go to the Grafana server at http://localhost:3000 and on the left-hand side, go to Home > Connections > Data sources.
```

Comment:
I am testing. I see the data sources. When I import the dashboards, all the panels say "no data". Looking more directly at the Thanos data source, it also says "no data".

<img width="917" alt="image" src="https://github.com/user-attachments/assets/9c5ee019-4472-4f3a-9b7b-960f66c45232">